app/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=bcgov-ccbc
+defaults.project=ccbc
+cli.executable=../../../.npm/_npx/a8388072043b4cbc/node_modules/@sentry/cli/bin/sentry-cli
```

Comment:
This last config item looks specific to your local setup, we probably want to remove that

Reply:
> A test asserting that the `postgraphile` function is called with the expected `postgraphileOptions` would be good. This would be somewhat similar to the test I just wrote for `authenticationPgSettings`, where you mock the config with different values (`NODE_ENV` production vs development, `SENTRY_ENVIRONMENT` is defined or not)

I think I might need some help with that if you have time at some point.